third_party/2and3/sqlalchemy/sql/elements.pyi

```diff
@@ -7,7 +9,38 @@ from .. import util
 
 class ClauseElement(Visitable): ...
 
-class ColumnElement(ColumnOperators, ClauseElement): ...
+class ColumnElement(ColumnOperators, ClauseElement):
+    __visit_name__ = 'column'  # type: Text
+    primary_key = False  # type: Any
+    foreign_keys = []  # type: Any
+    _label = None
+    _key_label = key = None
+    _alt_names = ()
```

Comment:
Guess I'll just annotate them with Any for now... I'll submit a PR in a few days.